src/components/OSCALControlGuidance.js

```diff
@@ -58,7 +58,7 @@ export default function OSCALControlGuidance(props) {
             ref={descriptionElementRef}
             tabIndex={-1}
           >
-            {props.prose}
+            {<OSCALMarkdownProse text = {props.prose}/>}
```

Comment:
Do you need the curly braces here?

```suggestion
            <OSCALMarkdownProse text = {props.prose}/>
```